piker/_daemon.py

```diff
@@ -447,3 +494,25 @@ async def maybe_open_emsd(
 
     ) as portal:
         yield portal
+
+
+# TODO: ideally we can start the tsdb "on demand" but it's
```

Comment:
Obvs coming down the road as part of what results from #247, left in as a guide.